tests/containers/skopeo.pm

```diff
@@ -15,6 +15,9 @@ use version_utils qw(is_transactional is_vmware);
 use transactional;
 use containers::common qw(install_packages);
 
+# Set a variable for test working directory
+our $workdir = '/tmp/test';
```

Comment:
i dont think this needs global variable ..

Reply:
It is used within 2 different subroutines. I tried a few different ways but besides hardcoding it, I found no other solution

Reply:
It has been already defined in the package, so you can use `my` instead of `our`

Reply:
I mean that is applicable for the whole package, you can see it in many modules